### PR TITLE
Add generated refdocs

### LINKED ISSUE #8387: Generated docs 404 - https://kubernetes.io/docs/reference/generated/kubelet/
**This is a...** 

- [ ] Feature Request
- [x] Bug Report

**Problem:**

Links to generated documents are not working. A few examples:

- https://kubernetes.io/docs/reference/generated/kubelet/
- https://kubernetes.io/docs/reference/generated/kube-proxy/
- https://kubernetes.io/docs/reference/generated/kube-controller-manager/

**Proposed Solution:**

**Page to Update:**
https://kubernetes.io/...

### PR DESCRIPTION
**Pull request by zacharysarah**
- fixes #8387 
- Add kubefed docs
- Add setup tools ref
- Add command line tools ref

### CODE REVIEW

**Comment by zacharysarah** [comment]
I don't get why `/command-line-tools-reference/` isn't appearing. Can someone else take a look, please?

File: content/en/docs/reference/command-line-tools-reference/_index.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+title: "Command line tools reference"
+weight: 60
+toc_hide: true
```
Comment (mistyhacks) [inline content/en/docs/reference/command-line-tools-reference/_index.md]: Maybe this is why it isn't showing up?

Comment (zacharysarah) [reply content/en/docs/reference/command-line-tools-reference/_index.md]: OH, it totally is--camel case, not snake case. Thanks!

**Comment by zacharysarah** [comment]
@mistyhacks Thanks--it's appearing properly now. https://deploy-preview-8405--kubernetes-io-master-staging.netlify.com/docs/reference/

**Comment by mistyhacks** [comment]
I did a spot check and it all looks fine, but I might be missing something.

/approve

**Comment by steveperry-53** [comment]
@zacharysarah, I've already done the Kubernetes API ref docs. You can remove those from this PR.

**Comment by steveperry-53** [comment]
Here's what I would suggest. Make one PR for [Command-line Tools Reference](https://v1-9.docs.kubernetes.io/docs/reference/generated/kubelet/). That would include these generated files:

* kubelet.md
* kube-apiserver.md
* kube-controller-manager.md
* kube-scheduler.md
* cloud-controller-manager.md
* federation-apiserver.md
* federation-controller-manager.md

As part of that PR, or in a followup PR, make sure those topics have a proper place in the TOC.
Then move on, in another PR, to the [Kubefed ref pages](https://v1-9.docs.kubernetes.io/docs/reference/generated/kubefed/).

**Comment by steveperry-53** [comment]
Also, the Kubernetes API docs don't belong under /content. As you can see in #8325, I have put them under /static, according to advice from Bjorn.

**Comment by zacharysarah** [comment]
This still needs an approval. @bradtopol?

**Comment by zacharysarah** [comment]
I'm closing this PR in favor of several smaller ones, per @steveperry-53.

**Pull request closed without merging**